shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -355,6 +355,34 @@ static FlutterAutofillType autofillTypeOf(NSDictionary* configuration) {
   return FlutterAutofillTypeNone;
 }
 
+static BOOL isPositionCloserToPoint(CGPoint point,
```

Comment:
Please, add docstring.  I'm not sure what this function should do when just reading the function signature.   I think you are asking is `point` closer to `position` or `otherPosition`?  We should rename position to match the datatype better since position seems to imply a point, how about "region"?

You seem to be doing distance math but you aren't using the distance formula.  If you want to see if a point is closer to one rect or another I'd just use the distance formula against the 4 corners.  (you could omit the sqrt for speed).